nifi-extension-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/ListenTrapSNMP.java

```diff
@@ -70,13 +76,14 @@
 @WritesAttribute(attribute = SNMPUtils.SNMP_PROP_PREFIX + "*", description = "Attributes retrieved from the SNMP response. It may include:"
         + " snmp$errorIndex, snmp$errorStatus, snmp$errorStatusText, snmp$nonRepeaters, snmp$requestID, snmp$type, snmp$variableBindings")
 @RequiresInstanceClassLoading
-public class ListenTrapSNMP extends AbstractSessionFactoryProcessor implements VerifiableProcessor {
+public class ListenTrapSNMP extends AbstractSessionFactoryProcessor implements VerifiableProcessor, ListenComponent {
 
     public static final PropertyDescriptor SNMP_MANAGER_PORT = new PropertyDescriptor.Builder()
             .name("SNMP Manager Port")
             .description("The port where the SNMP Manager listens to the incoming traps.")
             .required(true)
             .addValidator(StandardValidators.PORT_VALIDATOR)
+            .identifiesListenPort(TransportProtocol.UDP, "snmptrap")
```

Comment:
It seems like this should be `snmp`:

```suggestion
            .identifiesListenPort(TransportProtocol.UDP, "snmp")
```

Reply:
Thanks for the reference, I was originally thinking of SNMP in general, but I agree that `snmptrap` is more precise and aligns with the official assignments.